descriptors/ModuleDescriptor-template.json

```diff
@@ -231,6 +231,37 @@
         }
       ]
     },
+    {
+      "id": "acquisitions-unit-storage.units",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/acquisitions-unit-storage/units",
```

Comment:
@craigmcnally While reviewing this PR found out that described endpoint prefixes are different for the storage and business logic modules ( [Acquisitions API Listing](https://docs.google.com/spreadsheets/d/1se-a2owRfHLG5eaAZglx4vLvRClCKGp2wZCfD39ZJRY/view#gid=894044560&range=B71) ).

/acquisitions-unit-storage/units - for mod-orders-storage
/acquisitions-unit**s**/units - for mod-orders

I think this is a typo and endpoint naming should be changed to /acquisitions-unit**s**-storage/units. Permissions, interface name also should be updated according to this change.
P.S. I see the same issue for the already implemented endpoint `/order-storage/order-invoice-relns`. Should be used "/order**s**-storage/order-invoice-relns" instead.

Reply:
Ah good catch.  Yes, let's be consistent with singular vs plural.  So `orders-storage` and `acquisitions-units-storage`

Fortunately I don't think anything is using the `order-invoice-relns` APIs yet, so a breaking change there is likely harmless.